conmon-rs/server/src/rpc.rs

```diff
@@ -236,6 +247,7 @@ impl conmon::Server for Server {
                         &mut container_io,
                         &pidfile,
                         env_vars,
+                        vec![],
```

Comment:
non blocking nit: I'd prefer having this an Option type `None`.

Reply:
May an ask why? I would add an Option if the `None` and `Some(vec![])` value would represent different states. But in this case both values would do the same. (and `vec![]` does not allocate either)